sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_container_repository_client.py

```diff
@@ -22,7 +22,7 @@
 from .._models import (
```

Comment:
weird nit: it seems that the convention for naming async client files is more like `_container_repository_client_async.py` from my experience (even though the user won't see this). Is there a reason you did `_async_container...`?

Reply:
No reason, you're the first person to point this out too. I'll make a fix for that after this release, I think it will create a bunch of annoying merge conflicts if I do it now.